### PR TITLE
fix: use unzip to preserve permissions

### PR DESCRIPTION
**Pull request by viacheslav-rostovtsev**

### CODE REVIEW

**Review by dazuma — APPROVED** [review]

File: rules_ruby_gapic/docker_testing/Dockerfile [hunk 1]
```diff
@@ -32,13 +32,14 @@ RUN curl https://bazel.build/bazel-release.pub.gpg | apt-key add - \
...
-### Requirement: system has `git` to get the gapic repo
+### Requirement: system has `wget` to get the gapic-generator-ruby repo
+### Requirement: `tar` and `unzip` are both used to produce packaged built libraries
 ### Requirement: system has `make` to build ruby
 ### Requirement: system has `zlib` and `zlib-dev`, `libreadline` and `libreadline-dev`, `openssl` and `openssl-dev`, all needed for rubygems to work
 RUN apt-get update && apt-get install -y --no-install-recommends \
-        git \
         wget \
         tar \
+        unzip \
```
Comment (vam-google) [inline rules_ruby_gapic/docker_testing/Dockerfile:42]: Please strongly consider relying on https://github.com/googleapis/googleapis/blob/master/.kokoro/Dockerfile everywhere where you need a docker file, and remove the custom one.

**Review by vam-google — APPROVED** [review]
LGTM, but please strongly consider getting rid of the custom Dockerfile.

File: rules_ruby_gapic/docker_testing/Dockerfile (hunk 1 above)
Comment (viacheslav-rostovtsev) [reply rules_ruby_gapic/docker_testing/Dockerfile:42]: I'll consider that at the time I'll be moving the docker example to `bazel_examples`, a planned chore

Comment (viacheslav-rostovtsev) [reply rules_ruby_gapic/docker_testing/Dockerfile:42]: :+1:

**Pull request merged**